hadoop-tools/hadoop-distcp/src/main/java/org/apache/hadoop/tools/DistCpOptionSwitch.java

```diff
@@ -76,6 +76,14 @@
       new Option("delete", false, "Delete from target, " +
           "files missing in source. Delete is applicable only with update or overwrite options")),
 
+  /**
+   * When -delete option on, files in target that are missing from source
+   * will be delete by default. This allows the files to be
+   * moved to the trash
+   */
+  DELETE_USETRASH(DistCpConstants.CONF_LABEL_DELETE_MISSING_USETRASH,
+      new Option("useTrash", false, "Delete from target to the trash")),
```

Comment:
Or "Move deleted files into the user's trash directory in the destination filesystem"